votor-messages/src/vote.rs

```diff
@@ -28,6 +31,21 @@ pub enum Vote {
     Genesis(GenesisVote),
 }
 
+#[cfg(feature = "frozen-abi")]
+impl solana_frozen_abi::rand::prelude::Distribution<Vote>
+    for solana_frozen_abi::rand::distr::StandardUniform
+{
+    fn sample<R: solana_frozen_abi::rand::Rng + ?Sized>(&self, rng: &mut R) -> Vote {
+        match rng.random_range(0..5) {
+            0 => Vote::Notarize(rng.random()),
+            1 => Vote::Finalize(rng.random()),
+            2 => Vote::Skip(rng.random()),
+            3 => Vote::NotarizeFallback(rng.random()),
+            _ => Vote::SkipFallback(rng.random()),
```

Comment:
I would suggest we do an explicit arm for 4, and _ => unreachable!() to make sure this does not break in a glorious way when someone adds more options to Vote... Might be worth some extra checks that all variants are covered... this looks like a future footgun.